publish_trips.py

```diff
@@ -161,6 +174,10 @@ def handle_data(csv_text):
     data = classify_bike_type(data)
     # Add month/year columns
     data = populate_month_year(data)
+    # Add datetime column
+    data = datetime_creation(data)
```

Comment:
i know i established this pattern, but here's something i've been trying to get more consistent about: since each of the three functions above modify the rows in-place, it'd probably be better form to not reassign `data` after each call. so this instead:


```python
classify_bike_type(data)
# Add month/year columns
populate_month_year(data)
# Add datetime column
datetime_creation(data)
```

Reply:
Thanks for reminding me about this John. I forget that python will pass in a reference to mutable objects. Doing like you're saying would be a good habit for me to get into too.